MailShareExtension/Attachment/WeblocToHTMLAttachment.swift

```diff
@@ -40,20 +40,39 @@ struct WeblocToTextAttachment: TextAttachable {
                 return (nil, nil)
             }
 
-            guard let weblocData = NSData(contentsOf: webloc.url) else {
+            guard let weblocData = try? Data(contentsOf: webloc.url) else {
                 return (nil, nil)
             }
 
-            guard let parsedWebloc = try? PropertyListSerialization.propertyList(from: weblocData as Data,
+            guard let parsedWebloc = try? PropertyListSerialization.propertyList(from: weblocData,
                                                                                  options: [],
                                                                                  format: nil) as? NSDictionary else {
                 return (nil, nil)
             }
 
             let parsedURL = parsedWebloc["URL"] as? String
 
-            /// The `webloc` title is not useful for an email subject, discarding it
+            /// The `webloc` file name is generated, so not useful for an email subject, discarding it
             return TextAttachment(title: nil, body: parsedURL)
         }
     }
+
+    // MARK: HTMLAttachable protocol
+
+    var renderedHTML: String? {
+        get async {
+            guard let urlString = await textAttachment.body,
+                  let bodyUrl = URL(string: urlString) else {
+                return nil
+            }
+
+            let bodyAbsoluteUrl = bodyUrl.absoluteString
+            guard !bodyAbsoluteUrl.isEmpty else {
+                return nil
+            }
+
+            let finalHTML = "<div class=\"renderedHTML\"><a href=\"\(bodyAbsoluteUrl)\">" + bodyAbsoluteUrl + "</a></div>"
```

Comment:
Why do we need an HTML class renderedHTML?

Reply:
The idea was if we needed to debug an EML of a buggy email some day, we could see where this part of the DOM came from easily.

Reply:
I'm not a big fan of the idea of including unnecessary attributes, rather keep the code simple

Reply:
Sure, I agree also. I'll remove the extraneous class.